base/test-wget-debian.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+
+load 'assert'
+
+@test "ensure" {
+  run xx-apt remove --purge wget
+  run xx-wget ensure
+  assert_success
+}
+
+@test "install" {
+  run xx-apt remove --purge wget
+  run xx-wget install
+  assert_success
+}
+
+@test "ipv6" {
+  run xx-apt remove --purge wget
+  run xx-wget https://ipv6.google.com/
```

Comment:
This is expected to always fail I guess if busybox gets updated. Also, that's not exactly the current issue either I believe. The issue isn't that we should support ipv6-only hosts but for some reason, the fallback was not working.

Reply:
Yes this issue is not exactly the same, atm busybox wget doesn't have the ipv4 fallback implementation of GNU wget.